typewise_alert.test.py

```diff
@@ -5,7 +5,25 @@
 class TypewiseTest(unittest.TestCase):
   def test_infers_breach_as_per_limits(self):
     self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
-
+    self.assertTrue(typewise_alert.infer_breach(40, 10, 30) == 'TOO_HIGH')
+    self.assertTrue(typewise_alert.infer_breach(40, 20, 60) == 'NORMAL')
+    
+    
+  def test_breach_as_per_cooling_type(self):
+    self.assertTrue(typewise_alert.classify_temperature_breach('PASSIVE_COOLING',45) == 'TOO_HIGH')
+    self.assertTrue(typewise_alert.classify_temperature_breach('HI_ACTIVE_COOLING',-5) == 'TOO_LOW')
+    self.assertTrue(typewise_alert.classify_temperature_breach('MED_ACTIVE_COOLING',20) == 'NORMAL')
+    self.assertTrue(typewise_alert.classify_temperature_breach('',20) == 'Enter valid cooling type')
```

Comment:
Nice addition of error handling and test :)
Suggestion:
I guess you added the code first and then the test.
Try it the other way round. If you keep aside the implementation for a moment and focus on the test, you will look at it from a caller's perspective. In the valid cases, the caller receives a string that is a code (E.g., language-independent TOO_HIGH). But in error case, it gets a human-readable English string. If you write the test first, you are more likely to ask this question before the coding happens.
What do you feel?